.teamcity/kotlinlang/builds/BuildReferenceDocs.kt

```diff
@@ -1,64 +0,0 @@
-package kotlinlang.builds
```

Comment:
@Quattro8 it would be nice to announce changes for techwrighters, is it difficult keep it as archive project with comment before deletion?